content/installation/netcore/configuration/YAML-properties-netcore.md

```diff
@@ -88,8 +88,6 @@ Use the properties in this section to control security logging. These logs allow
 The following properties apply to any .NET Core agent-wide configurations. <!-- More words here... -->
 
   * **dotnet**:
-
-    
     * **enable_instrumentation_optimizations**: Indicate that the agent should allow CLR optimizations of JIT-compiled methods. 
     * **enable_jit_inlining**: Indicate that the agent should allow the CLR to inline methods that are not instrumented by Contrast.
     * **skip_profiler_check**: Indicate that the agent should not check for other profilers before starting.
```

Comment:
@m0nkey653 Should the `enable_chaining` property (a few lines down) be removed from this file, too?